package.json

```diff
@@ -36,9 +36,10 @@
   "bin": {
     "tapestry": "./bin/tapestry"
   },
+  "engineStrict": true,
```

Comment:
Tapestry won't run if you try and install in a node environment less than version 7.6